plugins/ODbgRegisterView/DialogEditFPU.cpp

```diff
@@ -76,7 +76,8 @@ long double readFloat(const QString &strInput, bool &ok) {
 }
 }
 
-DialogEditFPU::DialogEditFPU(QWidget *parent, Qt::WindowFlags f) : QDialog(parent, f), floatEntry_(new ODbgRegisterView::Float80Edit(this)), hexEntry_(new QLineEdit(this)) {
+DialogEditFPU::DialogEditFPU(QWidget *parent, Qt::WindowFlags f)
+	: QDialog(parent, f), floatEntry_(new ODbgRegisterView::Float80Edit(this)), hexEntry_(new QLineEdit(this)) {
```

Comment:
Maybe add a rule to format initialization lists in a column? I suggest, e.g. something like this:
```
	: QDialog(parent, f)
    , floatEntry_(new ODbgRegisterView::Float80Edit(this))
    , hexEntry_(new QLineEdit(this))
{
```
(Yes, with a deviation from the general rule about K&R braces, because this brace really gets lost in the parentheses otherwise.)

Reply:
Yes, the style guide will be updated to reflect how the code is actually formatted, good call. Usually, I address the parens getting lost by putting a newline after the initializer list, but I'm willing to take this under consideration as a better solution.